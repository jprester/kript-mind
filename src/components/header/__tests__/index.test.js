import React from 'react';
import { shallow } from 'enzyme';
import Header from '../index';

const wrapper = shallow((<Header />));

describe('index', () => {
  it('should render Header correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
