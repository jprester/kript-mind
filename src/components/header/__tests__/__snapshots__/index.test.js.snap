// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should render Header correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="main-container"
>
        <div
                className="header-content-logo"
        >
                <h1>
                        <a
                                className="logo-link"
                                href="./"
                        >
                                <img
                                        alt="logo"
                                        className="logo-image"
                                        src="./img/km_logo.png"
                                />
                                <span
                                        className="logo-text"
                                >
                                         
                                        <span
                                                className="logo-color-1"
                                        >
                                                Kript
                                        </span>
                                        <span
                                                className="logo-color-2"
                                        >
                                                Mind
                                        </span>
                                </span>
                        </a>
                        <span
                                className="title-label"
                        >
                                Beta
                        </span>
                </h1>
        </div>
        <div
                className="header-content-navigation"
        >
                <ul
                        className="navigation-links"
                >
                        <li>
                                <a
                                        href="./"
                                >
                                        About
                                </a>
                        </li>
                        <li>
                                <a
                                        href="./"
                                >
                                        Contacts
                                </a>
                        </li>
                </ul>
        </div>
</div>,
      "className": "header-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="header-content-logo"
>
            <h1>
                        <a
                                    className="logo-link"
                                    href="./"
                        >
                                    <img
                                                alt="logo"
                                                className="logo-image"
                                                src="./img/km_logo.png"
                                    />
                                    <span
                                                className="logo-text"
                                    >
                                                 
                                                <span
                                                            className="logo-color-1"
                                                >
                                                            Kript
                                                </span>
                                                <span
                                                            className="logo-color-2"
                                                >
                                                            Mind
                                                </span>
                                    </span>
                        </a>
                        <span
                                    className="title-label"
                        >
                                    Beta
                        </span>
            </h1>
</div>,
          <div
            className="header-content-navigation"
>
            <ul
                        className="navigation-links"
            >
                        <li>
                                    <a
                                                href="./"
                                    >
                                                About
                                    </a>
                        </li>
                        <li>
                                    <a
                                                href="./"
                                    >
                                                Contacts
                                    </a>
                        </li>
            </ul>
</div>,
        ],
        "className": "main-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1>
              <a
                            className="logo-link"
                            href="./"
              >
                            <img
                                          alt="logo"
                                          className="logo-image"
                                          src="./img/km_logo.png"
                            />
                            <span
                                          className="logo-text"
                            >
                                           
                                          <span
                                                        className="logo-color-1"
                                          >
                                                        Kript
                                          </span>
                                          <span
                                                        className="logo-color-2"
                                          >
                                                        Mind
                                          </span>
                            </span>
              </a>
              <span
                            className="title-label"
              >
                            Beta
              </span>
</h1>,
            "className": "header-content-logo",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <a
                  className="logo-link"
                  href="./"
>
                  <img
                                    alt="logo"
                                    className="logo-image"
                                    src="./img/km_logo.png"
                  />
                  <span
                                    className="logo-text"
                  >
                                     
                                    <span
                                                      className="logo-color-1"
                                    >
                                                      Kript
                                    </span>
                                    <span
                                                      className="logo-color-2"
                                    >
                                                      Mind
                                    </span>
                  </span>
</a>,
                <span
                  className="title-label"
>
                  Beta
</span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <img
                      alt="logo"
                      className="logo-image"
                      src="./img/km_logo.png"
/>,
                    <span
                      className="logo-text"
>
                       
                      <span
                                            className="logo-color-1"
                      >
                                            Kript
                      </span>
                      <span
                                            className="logo-color-2"
                      >
                                            Mind
                      </span>
</span>,
                  ],
                  "className": "logo-link",
                  "href": "./",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "alt": "logo",
                      "className": "logo-image",
                      "src": "./img/km_logo.png",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": "img",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        " ",
                        <span
                          className="logo-color-1"
>
                          Kript
</span>,
                        <span
                          className="logo-color-2"
>
                          Mind
</span>,
                      ],
                      "className": "logo-text",
                    },
                    "ref": null,
                    "rendered": Array [
                      " ",
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Kript",
                          "className": "logo-color-1",
                        },
                        "ref": null,
                        "rendered": "Kript",
                        "type": "span",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Mind",
                          "className": "logo-color-2",
                        },
                        "ref": null,
                        "rendered": "Mind",
                        "type": "span",
                      },
                    ],
                    "type": "span",
                  },
                ],
                "type": "a",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Beta",
                  "className": "title-label",
                },
                "ref": null,
                "rendered": "Beta",
                "type": "span",
              },
            ],
            "type": "h1",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <ul
              className="navigation-links"
>
              <li>
                            <a
                                          href="./"
                            >
                                          About
                            </a>
              </li>
              <li>
                            <a
                                          href="./"
                            >
                                          Contacts
                            </a>
              </li>
</ul>,
            "className": "header-content-navigation",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li>
                  <a
                                    href="./"
                  >
                                    About
                  </a>
</li>,
                <li>
                  <a
                                    href="./"
                  >
                                    Contacts
                  </a>
</li>,
              ],
              "className": "navigation-links",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="./"
>
                    About
</a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "About",
                    "href": "./",
                  },
                  "ref": null,
                  "rendered": "About",
                  "type": "a",
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <a
                    href="./"
>
                    Contacts
</a>,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Contacts",
                    "href": "./",
                  },
                  "ref": null,
                  "rendered": "Contacts",
                  "type": "a",
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="main-container"
>
          <div
                    className="header-content-logo"
          >
                    <h1>
                              <a
                                        className="logo-link"
                                        href="./"
                              >
                                        <img
                                                  alt="logo"
                                                  className="logo-image"
                                                  src="./img/km_logo.png"
                                        />
                                        <span
                                                  className="logo-text"
                                        >
                                                   
                                                  <span
                                                            className="logo-color-1"
                                                  >
                                                            Kript
                                                  </span>
                                                  <span
                                                            className="logo-color-2"
                                                  >
                                                            Mind
                                                  </span>
                                        </span>
                              </a>
                              <span
                                        className="title-label"
                              >
                                        Beta
                              </span>
                    </h1>
          </div>
          <div
                    className="header-content-navigation"
          >
                    <ul
                              className="navigation-links"
                    >
                              <li>
                                        <a
                                                  href="./"
                                        >
                                                  About
                                        </a>
                              </li>
                              <li>
                                        <a
                                                  href="./"
                                        >
                                                  Contacts
                                        </a>
                              </li>
                    </ul>
          </div>
</div>,
        "className": "header-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="header-content-logo"
>
              <h1>
                            <a
                                          className="logo-link"
                                          href="./"
                            >
                                          <img
                                                        alt="logo"
                                                        className="logo-image"
                                                        src="./img/km_logo.png"
                                          />
                                          <span
                                                        className="logo-text"
                                          >
                                                         
                                                        <span
                                                                      className="logo-color-1"
                                                        >
                                                                      Kript
                                                        </span>
                                                        <span
                                                                      className="logo-color-2"
                                                        >
                                                                      Mind
                                                        </span>
                                          </span>
                            </a>
                            <span
                                          className="title-label"
                            >
                                          Beta
                            </span>
              </h1>
</div>,
            <div
              className="header-content-navigation"
>
              <ul
                            className="navigation-links"
              >
                            <li>
                                          <a
                                                        href="./"
                                          >
                                                        About
                                          </a>
                            </li>
                            <li>
                                          <a
                                                        href="./"
                                          >
                                                        Contacts
                                          </a>
                            </li>
              </ul>
</div>,
          ],
          "className": "main-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <h1>
                <a
                                className="logo-link"
                                href="./"
                >
                                <img
                                                alt="logo"
                                                className="logo-image"
                                                src="./img/km_logo.png"
                                />
                                <span
                                                className="logo-text"
                                >
                                                 
                                                <span
                                                                className="logo-color-1"
                                                >
                                                                Kript
                                                </span>
                                                <span
                                                                className="logo-color-2"
                                                >
                                                                Mind
                                                </span>
                                </span>
                </a>
                <span
                                className="title-label"
                >
                                Beta
                </span>
</h1>,
              "className": "header-content-logo",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <a
                    className="logo-link"
                    href="./"
>
                    <img
                                        alt="logo"
                                        className="logo-image"
                                        src="./img/km_logo.png"
                    />
                    <span
                                        className="logo-text"
                    >
                                         
                                        <span
                                                            className="logo-color-1"
                                        >
                                                            Kript
                                        </span>
                                        <span
                                                            className="logo-color-2"
                                        >
                                                            Mind
                                        </span>
                    </span>
</a>,
                  <span
                    className="title-label"
>
                    Beta
</span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <img
                        alt="logo"
                        className="logo-image"
                        src="./img/km_logo.png"
/>,
                      <span
                        className="logo-text"
>
                         
                        <span
                                                className="logo-color-1"
                        >
                                                Kript
                        </span>
                        <span
                                                className="logo-color-2"
                        >
                                                Mind
                        </span>
</span>,
                    ],
                    "className": "logo-link",
                    "href": "./",
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "alt": "logo",
                        "className": "logo-image",
                        "src": "./img/km_logo.png",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": "img",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          " ",
                          <span
                            className="logo-color-1"
>
                            Kript
</span>,
                          <span
                            className="logo-color-2"
>
                            Mind
</span>,
                        ],
                        "className": "logo-text",
                      },
                      "ref": null,
                      "rendered": Array [
                        " ",
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "Kript",
                            "className": "logo-color-1",
                          },
                          "ref": null,
                          "rendered": "Kript",
                          "type": "span",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "Mind",
                            "className": "logo-color-2",
                          },
                          "ref": null,
                          "rendered": "Mind",
                          "type": "span",
                        },
                      ],
                      "type": "span",
                    },
                  ],
                  "type": "a",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Beta",
                    "className": "title-label",
                  },
                  "ref": null,
                  "rendered": "Beta",
                  "type": "span",
                },
              ],
              "type": "h1",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ul
                className="navigation-links"
>
                <li>
                                <a
                                                href="./"
                                >
                                                About
                                </a>
                </li>
                <li>
                                <a
                                                href="./"
                                >
                                                Contacts
                                </a>
                </li>
</ul>,
              "className": "header-content-navigation",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <li>
                    <a
                                        href="./"
                    >
                                        About
                    </a>
</li>,
                  <li>
                    <a
                                        href="./"
                    >
                                        Contacts
                    </a>
</li>,
                ],
                "className": "navigation-links",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      href="./"
>
                      About
</a>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "About",
                      "href": "./",
                    },
                    "ref": null,
                    "rendered": "About",
                    "type": "a",
                  },
                  "type": "li",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <a
                      href="./"
>
                      Contacts
</a>,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Contacts",
                      "href": "./",
                    },
                    "ref": null,
                    "rendered": "Contacts",
                    "type": "a",
                  },
                  "type": "li",
                },
              ],
              "type": "ul",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
